hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/configuration/FlinkOptions.java

```diff
@@ -275,7 +275,7 @@ private FlinkOptions() {
   public static final ConfigOption<Boolean> UTC_TIMEZONE = ConfigOptions
       .key("read.utc-timezone")
       .booleanType()
-      .defaultValue(true)
+      .defaultValue(false)
```

Comment:
Thanks, can we try to add a IT in `ITTestHoodieDataSource` ?

Reply:
I have  added some tests.

Reply:
Guess we need to keep default value as true.